packages/quick_actions/android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="io.flutter.plugins.quickactions">
+    <application>
```

Comment:
you don't want to do this in this file, you should do it in the examples directory

Reply:
Yeah this was left over from an improper thing I did on the other one. I know where it goes now :)

Reply:
Also feel empowered to edit http://flutter.dev/go/android-plugin-migration if there are confusing things in that doc as well.